app/full_text_search.py

```diff
@@ -22,6 +22,15 @@
 
 # The index name for full text search
 PERSON_FULL_TEXT_INDEX_NAME = 'person_information'
+PERSON_LOCATION_FULL_TEXT_INDEX_NAME = 'person_location_information'
```

Comment:
Can you add a short comment which explains the difference between PERSON_FULL_TEXT_INDEX_NAME and PERSON_LOCATION_FULL_TEXT_INDEX_NAME?